kubernetes/omsagent.yaml

```diff
@@ -445,59 +453,65 @@ spec:
             periodSeconds: 60
             timeoutSeconds: 15
 #Only in sidecar scraping mode
-        - name: omsagent-prometheus
-          image: "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod06112021"
-          imagePullPolicy: IfNotPresent
-          resources:
-            limits:
-              cpu: 500m
-              memory: 1Gi
-            requests:
-              cpu: 75m
-              memory: 225Mi
-          env:
-            # azure devops pipeline uses AKS_RESOURCE_ID and AKS_REGION hence ensure to uncomment these
-            - name: AKS_RESOURCE_ID
-              value: "VALUE_AKS_RESOURCE_ID_VALUE"
-            - name: AKS_REGION
-              value: "VALUE_AKS_RESOURCE_REGION_VALUE"
-            #Uncomment below two lines for ACS clusters and set the cluster names manually. Also comment out the above two lines for ACS clusters
-            #- name: ACS_RESOURCE_NAME
-            #  value: "my_acs_cluster_name"
-            - name: CONTAINER_TYPE
-              value: "PrometheusSidecar"
-            - name: CONTROLLER_TYPE
-              value: "DaemonSet"
-            - name: NODE_IP
-              valueFrom:
-                fieldRef:
-                  fieldPath: status.hostIP
-            # Update this with the user assigned msi client id for omsagent
-            - name: USER_ASSIGNED_IDENTITY_CLIENT_ID
-              value: ""
-          securityContext:
-            privileged: true
-          volumeMounts:
-            - mountPath: /etc/kubernetes/host
-              name: azure-json-path
-            - mountPath: /etc/omsagent-secret
-              name: omsagent-secret
-              readOnly: true
-            - mountPath: /etc/config/settings
-              name: settings-vol-config
-              readOnly: true
-            - mountPath: /etc/config/osm-settings
-              name: osm-settings-vol-config
-              readOnly: true
-          livenessProbe:
-            exec:
-              command:
-                - /bin/bash
-                - -c
-                - /opt/livenessprobe.sh
-            initialDelaySeconds: 60
-            periodSeconds: 60
-            timeoutSeconds: 15
+        # - name: omsagent-prometheus
```

Comment:
since this can be controlled via SIDECAR_SCRAPING_ENABLED, why do we need to comment this?

Reply:
This creates a sidecar container. the goal is to keep it consistent with AKS-RP. Commenting it now, will be uncommenting it when we start rolling out to regions,